### DISCUSSION TITLE
difference between ubuntu 20.04 and ubuntu 22.04

### DISCUSSION THREAD

**Discussion by alain38500** [Jakeler/ble-serial#77] [Q&A]
Hi

We are having a problem receiving characters on ble-serial. we work under linux 20.04 and linux 22.04 environment ( or probleme with  the bluetooth chip  or with the hci drivers ?) . We use ble-serial to connect a termios serial terminal to an embedded system shell using an RN4871.

So far we have identified a problem on linux 20.04 PCs. when the embedded system sends a lot of characters with a high baud rate, the RN4871 module raises its CTS, and blocks the sending, at this moment no more characters are transmitted from the embedded to the linux, you have to cut off ble-serial to be able to re-open the channel.
important: we have no problem with linux in 22.04, even with a high baud rate on RN4718, 921600.

**The command we use:** ble-serial -d 04:91:62:AE:C0:91 -w 49535343-1e4d-4bd9-ba61-23c647249616 -r 49535343-1e4d-4bd9-ba61-23c647249616 -b - l test.log -m 60

regards

**Comment by Jakeler** [comment]
I don't have this hardware and am not aware of such problems with Ubuntu 20.04, but would assume it has something to do with the bluez version, maybe even kernel (hw drivers). 
Also you could run ble-serial with `-v` or `-vv` and see if there is a difference depending on the OS.
